src/vmm/src/guest_config/aarch64_templates/g3g2.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use arch::aarch64::regs::{ID_AA64ISAR0_EL1, ID_AA64ISAR1_EL1, ID_AA64MMFR2_EL1, ID_AA64PFR0_EL1};
+
+use super::{CpuTemplateAarch64, RegisterValue};
+
+/// Aarch64 template to mask graviton3 as graviton2
+pub fn g3g2() -> CpuTemplateAarch64 {
+    CpuTemplateAarch64(
+        vec![
+            (
+                // Disable dit CPU feature.
```

Comment:
It would be great if we could provide also link to the ARM developer spec, to make them easier to find.

Reply:
Also let's specify for all of them the bits and the register they are placed in, the way you do below:
```
sha3 occupies bits [35:32] in ID_AA64ISAR0_EL1
```

Reply:
done